packages/browser-wallet/src/popup/popupX/pages/Onboarding/IdSubmitted.tsx

```diff
@@ -21,7 +20,16 @@ export default function IdSubmitted() {
             <Page.Top heading={t('yourId')} />
             <Page.Main>
                 <Text.Capture>{t('idSubmitInfo')}</Text.Capture>
-                <IdCard rowsIdInfo={rowsIdInfo} idProviderName="TODO" identityName="TODO" />
+                <IdCard title="Identity 1" subtitle="Submitted to NotaBene">
```

Comment:
Is this (i.e. "Identity 1", "Submitted by NotaBene") accurate?

Reply:
These are again just placeholder text until we implement the onboarding flow.